src/sources/dnstap/mod.rs

```diff
@@ -195,8 +258,31 @@ impl FrameHandler for DnstapFrameHandler {
 
         let mut log_event = LogEvent::default();
 
+        if self.log_namespace == LogNamespace::Vector {
```

Comment:
There is a `insert_standard_vector_source_metadata` that will insert the `ingest_timestamp` and `source_type` for you.